backend/main.py

```
@@ -1,369 +1,407 @@
-          summary="Submit a direct help request",
-          tags=["Intake Agent"])
-async def submit_direct_request(payload: schemas.DirectHelpRequestPayload = Body(...)):
-    """
-    Allows users to submit a help request directly through the application.
-    The IntakeAgent will process this request.
-    """
-    logger.info(f"API /api/v1/request/direct received payload: {payload.model_dump_json(indent=2)}")
+async def read_root(): return {"status": "SOS Nairobi Backend is running"}
+
+@app.post("/api/v1/request/direct", response_model=schemas.GenericResponse, tags=["Intake Agent"])
+async def submit_direct_request(request: Request, payload: schemas.DirectHelpRequestPayload = Body(...)):
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)